lightning-liquidity/Cargo.toml

```diff
@@ -14,8 +14,9 @@ categories = ["cryptography::cryptocurrencies"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [features]
-default = ["std"]
+default = ["std", "time"]
```

Comment:
Disabling `SystemTIme::now` without disabling all of std: couldn't this be realized by just passing in another time provider than the std one, without requiring this flag?